Software/lib/GNSE_BSP/GNSE_bsp_serial.h

```diff
@@ -138,7 +128,18 @@ extern DMA_HandleTypeDef GNSE_BSP_hdma_tx;
 #define EXT_SENSOR_I2C2_SCL_SDA_AF                 GPIO_AF4_I2C2
 
 #define FLASH_SPI SPI1
-#define FLASH_SPI_BAUDRATE            SPI_BAUDRATEPRESCALER_256
+/**
+ * Speed of the SPI interface depends on both the system clock speed (48 MHz required for LoRa) and a prescaler.
+ * Baud rate can be one of the following values:
+ * 24MHz @48MHz clock: SPI_BAUDRATEPRESCALER_2
+ * 12MHz @48MHz clock: SPI_BAUDRATEPRESCALER_4
+ * 6MHz @48MHz clock: SPI_BAUDRATEPRESCALER_8
+ * 3MHz @48MHz clock: SPI_BAUDRATEPRESCALER_16
+ * 1.5MHz @48MHz clock: SPI_BAUDRATEPRESCALER_32
+ * 375kHz @48MHz clock: SPI_BAUDRATEPRESCALER_128
+ * 187.5kHz @48MHz clock: SPI_BAUDRATEPRESCALER_256
+ */
```

Comment:
I don't think this is necessary, let's remove it as well and add a small comment on the current default, same as the I2C